test/unit/models/forms/LiabilityDetailsFormTest.scala

```diff
@@ -68,10 +68,17 @@ class LiabilityDetailsFormTest extends ModelsBaseSpec {
     "officerCommodityCode" -> Seq("0100000000")
   )
 
+  private val emptyParams = params.map {
```

Comment:
Same with the others as it can just be `params.view.mapValues(_ => Seq("")).toMap`. Thanks